src/test/app/Offer_test.cpp

```diff
@@ -4397,15 +4394,19 @@ class Offer_test : public beast::unit_test::suite
         env.require (offers (gw, 0));
     }
 
-    void testTickSize (std::initializer_list<uint256> fs)
+    void testTickSize (FeatureBitset features)
     {
         testcase ("Tick Size");
 
         using namespace jtx;
 
+        // Disable featureTickSize for the moment.  We'll re-enable it later.
+        FeatureBitset const featuresWoTickSize {
+            features & ~with_only_features(featureTickSize)};
+
         // Try to set tick size without enabling feature
         {
-            Env env {*this, with_features(fs)};
+            Env env {*this, featuresWoTickSize};
```

Comment:
This is the only place where `featuresWoTickSize` is used. I'd remove the variable and replace this with `features & ~with_only_features(featureTickSize)`